### ISSUE TITLE
"Media Files" check not showing up

### ISSUE THREAD

**Issue by lucrus73** [wp-sync-db/wp-sync-db-media-files#31]
I've installed this extension along with wp-sync-db in three different WP instances. None of them show the "Media Files" checkbox. I suspect I'm not installing this the right way, but installation is not documented. Isn't it enouhg to download and unzip into wp-content/plugins?

**Comment by ghost** [comment]
There seems to be some inline CSS hiding the check box. got in the the plugin folder for wp-sync-db and add this to the style.css

.media-files-options {
  display: block !important;
}

**Comment by lucrus73** [comment]
You're right, thanks for the hint. However I believe that when someone inlined a `display: none` he did that for a reason, and overriding it could lead to more trouble.
So I've debugged the js and [I've found this is a feature](https://github.com/wp-sync-db/wp-sync-db-media-files/blob/c14b6128e506006a8502684c66690f2e2129483a/asset/js/script.js#L52) rather than a bug, albeit it seems to be undocumented.

From what I get in the code, the inteded behavior is: if we are migrating with a remote connection (e.g. not with a backup file) and the remote connection is ok, and the remote plugin is the same version as the local, then show the media files checkbox, otherwise hide it.

I've tried entering the correct connection informations and the checkbox does show up. Problem solved, thanks.
